cmake/TorchPackage.cmake

```diff
@@ -1,5 +1,26 @@
 # -*- cmake -*-
 
+MACRO(ADD_TORCH_LIBRARY package type src)
+IF ("${type}" STREQUAL "STATIC")
+  if ("${src}" MATCHES "cu$" OR "${src}" MATCHES "cu;")
+    CUDA_ADD_LIBRARY(${package} STATIC ${src})
+  else()
+    ADD_LIBRARY(${package} STATIC ${src})
+  endif()
+ELSE()
+  IF (BUILD_STATIC OR "$ENV{TH_ONLY_STATIC}" STREQUAL "YES")
+    ADD_LIBRARY(${package} ${type} "")
```

Comment:
Hmm it looks like I messed something up during rebasing. Let me double check this.